packages/gatsby-transformer-remark/src/__tests__/extend-node.js

```diff
@@ -85,7 +85,13 @@ Where oh where is my little pony?
           types: [{ name: `MarkdownRemark` }],
         }
       ).then(result => {
-        expect(_.isString(result.data.listNode.excerpt))
+        try {
+          expect(_.isString(result.data.listNode[0].excerpt)).toBeTruthy()
```

Comment:
maybe using `.toMatchSnapshot()` instead `.toBeTruthy()` would make sense here? so we actually see excerpt and verify that produced excerpt isn't being changed with other changes?

Reply:
the expected excerpt is `''`. maybe i can just test it to match that? i felt like adding a snapshot for that might not be needed

Reply:
Oh, definitely not! I thought there'd be some more data, but how about just

```
const excerpt = _.get(result, 'data.listNode[0].excerpt');

expect(excerpt).toBe('');
```

Reply:
also, actually - `listNode` shouldn't be an array - it should be object - not sure what's going on there

Reply:
hmmm, why is this a list https://github.com/gatsbyjs/gatsby/blob/4602cbaa320654a6e7c75d7bf4e781db58e62bd0/packages/gatsby-transformer-remark/src/__tests__/extend-node.js#L22 :S

Reply:
ah, so we could actually test more samples
so we should probably use more sample markdown files to actually make use of that

but as @DSchau mentioned - I don't think that default excerpt should be empty string - if it is either excerpt or test is broken and doesn't do what it should?

Reply:
well i am not sure anymore what this test is supposed to test. i understood that it tested the default value, which i thought it was supposed to be `''`. :S i am a bit lost now :D

Reply:
actually that test looks pretty similar to this one https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-transformer-remark/src/__tests__/gatsby-node.js#L30 only with a few more checks in it.